app/frontend/stores/jurisdiction-store.ts

```diff
@@ -84,18 +84,39 @@ export const JurisdictionStoreModel = types
         self.environment.api.searchJurisdictions({
           query: self.query,
           sort: self.sort,
-          page: opts?.page ?? self.currentPage,
-          perPage: opts?.countPerPage ?? self.countPerPage,
           filters: {
             submissionInboxSetUp,
           },
         })
       )
       if (response.ok) {
-        self.mergeUpdateAll(response.data.data, "jurisdictionMap")
-        self.tableJurisdictions = cast(response.data.data.map((jurisdiction) => jurisdiction.id))
-        self.currentPage = opts?.page ?? self.currentPage
-        self.countPerPage = opts?.countPerPage ?? self.countPerPage
+        const camelizedData = keysToCamelCase(response.data.data) as IJurisdiction[]
+        let sortedData = camelizedData
+
+        if (self.sort && self.sort.field === "reverse_qualified_name") {
+          sortedData = camelizedData.sort((a: any, b: any) => {
+            const nameA = a.reverseQualifiedName?.toUpperCase() || ""
```

Comment:
Sort is something that happens at the searchkick level. it's probably best to only have one kind of sorting.